integration/ct_integration_test.go

```diff
@@ -124,3 +164,28 @@ func certsFromPEM(data []byte) []ct.ASN1Cert {
 	}
 	return chain
 }
+
+func getChain(path string) ([]ct.ASN1Cert, error) {
+	certdata, err := ioutil.ReadFile(filepath.Join(*testdata, path))
+	if err != nil {
+		return nil, fmt.Errorf("Failed to load certificate: %v", err)
+	}
+	return certsFromPEM(certdata), nil
+}
+
+func awaitTreeSize(ctx context.Context, logClient *client.LogClient, size uint64, exact bool) (*ct.SignedTreeHead, error) {
```

Comment:
I'd also say it doesn't belong here -- the integration test only exercises the 6962 HTTP API, with no knowledge (deliberately) of anything behind that API.